src/Apps/Order2/Routes/Checkout/Components/OfferStep/Order2OfferStep.tsx

```diff
@@ -189,11 +191,9 @@ export const Order2OfferStep: React.FC<Order2OfferStepProps> = ({ order }) => {
         px={[2, 4]}
         hidden={currentStep !== CheckoutStepState.COMPLETED}
       >
-        <Order2OfferCompletedView
-          order={orderData}
-          offerAmount={submittedOfferAmount}
-          offerNote={submittedOfferNote}
-        />
+        {completedViewProps && (
+          <Order2OfferCompletedView {...completedViewProps} />
+        )}
```

Comment:
It would be good to have a stronger opinion and pattern on how the step state vs props work together. It varies across steps (maybe for good reason) but hopefully would work for any case. I think the `step.state` is still necessary because a step could be completed in the sense of having saved data, but currently be getting edited or even in the future if a user has gone back.